source/math/Quaternion.ooc

```diff
@@ -33,7 +33,7 @@ Quaternion: cover {
 	// z = this y
 	// w = this z
 
-	inverse ::= This new(this w, -this x, -this y, -this z)
+	inverse ::= This new(this w / (this norm * this norm), -this x / (this norm * this norm), -this y / (this norm * this norm), - this z / (this norm * this norm))
```

Comment:
It should yes, however, ```-``` along with a few other operators were "temporarily" disabled early on due to being both unary and binary, so we were going to catch those kind of rules in the parsing phase instead. However, we decided to rewrite magic from the ground up, so it was never fixed in this version of magic.